adapters/kosovo.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * This code is responsible for implementing all methods related to fetching
+ * and returning data for a Kosovo sources (HTML page)
+ */
+'use strict';
+
+const moment = require('moment-timezone');
+const request = require('request-promise-native');
+const cheerio = require('cheerio');
+
+const log = require('../lib/logger');
+
+const openaq_parameters = ['pm25', 'pm10', 'no2', 'so2', 'o3', 'co', 'bc'];
```

Comment:
Please import these similar to https://github.com/openaq/openaq-fetch/blob/develop/adapters/airnow-http.js#L5